docs/source/tutorials/sveltekit.mdx

```diff
@@ -0,0 +1,431 @@
+  kit: {
+    adapter: adapter({
+      // default options are shown
+      out: "build",
+      precompress: false,
+      envPrefix: "",
+      polyfill: true,
+    }),
+  },
+  preprocess: vitePreprocess(),
+};
+export default config;
+```
+
+### Step 3: Install and configure Transformers.js
+
+You can install Transformers.js from [NPM](https://www.npmjs.com/package/@xenova/transformers) with the following command:
+
+```bash
+npm i @xenova/transformers
+```
+
+`./src/lib/server/pipeline.js` - to handle the construction of our pipeline.
+
+```js
+import { pipeline } from "@xenova/transformers";
+
+// Use the Singleton pattern to enable lazy construction of the pipeline.
+// NOTE: We wrap the class in a function to prevent code duplication (see below).
+const P = () =>
+  class PipelineSingleton {
+    static task = "text-classification";
+    static model = "Xenova/distilbert-base-uncased-finetuned-sst-2-english";
+    static instance = null;
+
+    static async getInstance(progress_callback = null) {
+      if (this.instance === null) {
+        this.instance = pipeline(this.task, this.model, {
+          progress_callback,
+        });
+      }
+      return this.instance;
+    }
+  };
+
+let PipelineSingleton;
+if (process.env.NODE_ENV !== "production") {
+  // When running in development mode, attach the pipeline to the
+  // global object so that it's preserved between hot reloads.
+  // For more information, see https://vercel.com/guides/nextjs-prisma-postgres
+  if (!global.PipelineSingleton) {
+    global.PipelineSingleton = P();
+  }
+  PipelineSingleton = global.PipelineSingleton;
+} else {
+  PipelineSingleton = P();
+}
+export default PipelineSingleton;
+```
+
+Our only API route will be a `GET` request to `/classify` that takes a `text` parameter and returns the classification result.
+This is defined in `./src/routes/classify/+server.js`.
+
+```jsx
+import { json } from "@sveltejs/kit";
+import PipelineSingleton from "$lib/server/pipeline.js";
+
+export async function GET({ url }) {
+  const text = url.searchParams.get("text");
+  if (!text) {
+    return json(
+      {
+        error: "Missing text parameter",
+      },
+      { status: 400 }
+    );
+  }
+  // Get the classification pipeline. When called for the first time,
+  // this will load the pipeline and cache it for future use.
+  const classifier = await PipelineSingleton.getInstance();
+
+  // Actually perform the classification
+  const result = await classifier(text);
+
+  return json(result);
+}
+```
+
+### Step 4: Design the user interface
+
+We'll define the main interface and logic for our application on the single page component `./src/routes/+page.svelte` that maps to `/`.
+The main difference with the client-side inference is that we'll be using the `fetch` API to make requests to our API route handler, rather than using a Web Worker.
+
+```jsx
+<script>
+	let result = null;
+	let ready = null;
+
+	async function classify(text) {
+		if (!text) return;
```

Comment:
Tabs vs. spaces here too (prefer 2-spaces for example code)

Reply:
defiantly spaces, not sure why sveltekit create project is coming with tabs 🤦